srcpkgs/gowall/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'gowall'
+pkgname=gowall
+version=0.1.9
+revision=1
+build_style=go
+go_import_path="github.com/Achno/${pkgname}"
+short_desc="Gowall is a tool to convert an image to any color-scheme/pallete"
+maintainer="elbachir-one <bachiralfa@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Achno/gowall"
+distfiles="https://github.com/Achno/gowall/archive/refs/tags/v${version}.tar.gz"
+checksum=a01448fe9e81f1abb68759599839b556694fecbfd60227de4d3cd1231eb9e02e
+
+do_build() {
+	go build -o gowall
+}
+
+do_install() {
+	install -Dm755 gowall "$DESTDIR"/usr/bin/gowall
+}
```

Comment:
that's unnecessary, the go build style does that automatically and handles cross compilation for you

Reply:
Ok, Thank you.